_pages/release-strategies.md

```diff
@@ -6,106 +6,114 @@ sticky_sidenav: true
 
 Releasing software is hard. Over time, we have developed strategies that can help reduce risk and uncertainity in the release process.
 
-## Small, Iterative Releases
+## Small, iterative releases
 
 We recommend on all projects to get into the habit of releasing software as early and often as possible. The first
-iteration of your software should not aim to have full functionality; a better goal for the very first release is to have as little
-functionality as possible.
+iteration of your software should not aim to have full functionality; a better goal for the very first release is to have as [little
+functionality as possible](https://18f.gsa.gov/2017/01/11/the-best-way-to-build-big-is-to-start-small/#start-small-make-it-end-to-end). 
+A small-and-quick first release can help you [test your deployment pipeline and environments, while setting expectations for a rapid,
+iterative release cadence](https://blog.thepete.net/blog/2019/10/04/hello-production/).
```

Comment:
i think we need to build out the recommendation FOR small iterative releases, but that was outside the scope of this original write-up.

Reply:
I think we've written so much on it before a link (or links) is fine.